Author: Modestas Vainius <modax@debian.org>
Description: do not export QtUiTools symbols in the libprocessui dynamic symbol table
Forwarded: no
Origin: vendor
Last-Update: 2010-04-30

--- a/libs/ksysguard/processui/CMakeLists.txt
+++ b/libs/ksysguard/processui/CMakeLists.txt
@@ -22,6 +22,9 @@
 
 kde4_add_library(processui SHARED ${processui_LIB_SRCS})
 
+# Do not export QtUiTools internal symbols
+set_target_properties(processui PROPERTIES LINK_FLAGS_DEBIAN "-Wl,--exclude-libs -Wl,libQtUiTools.a")
+
 if(X11_XRes_FOUND)
   target_link_libraries(processui ${X11_XRes_LIB} ${X11_LIBRARIES})
   include_directories(${X11_XRes_INCLUDE_PATH})
